### PR TITLE
Added rating to endings, added some grammar, fixed some case issues

### PR DESCRIPTION
**Pull request by LaTueur**

### CODE REVIEW

**Comment by ben-bay** [comment]
This looks great, before I test it could you look at the conflicts that recently emerged? Also hit up the CHANGELOG plz :)

**Comment by LaTueur** [comment]
Done.

File: story/grammars/fantasy_rules.json [hunk 1]
```diff
@@ -34,14 +34,14 @@
...
+    "attacked" : ["under attack", "being invaded", "in danger", "being surrounded", "being sieged"],
     "noble_one" : ["One morning, you are awakened by one of your #subordinate.s#. #he_she.capitalize# tells you that your #nice_home# is #attacked#. You look out #a_the# window and see"],
 
     "element" : ["fire", "water", "earth", "air"],
     "more_element" : ["the underworld", "the sea", "the heavens"],
     "celestial" : ["the sun", "the moon", "the stars"],
-    "domain" : ["#element#", "#more_element#", "#celestial#", "harvest", "fertility", "war", "good fortune", "thresholds", "love"],
-    "god" : ["god", "deity"],
+    "domain" : ["#element#", "#more_element#", "#celestial#", "harvest", "fertility", "war", "good fortune", "thresholds", "love", "wisdom", "#profession.s#"],
+    "god" : ["god", "deity", "godess"],
```
Comment (CaoRuiming) [inline story/grammars/fantasy_rules.json]: Minor spelling error: `godess` should be `goddess`

Comment (LaTueur) [reply story/grammars/fantasy_rules.json]: Thanks, corrected.

**Pull request merged**